ipaclient/install/client.py

```diff
@@ -2345,6 +2339,74 @@ def update_ipa_nssdb():
                                    (nickname, sys_db.secdir, e))
 
 
+def sync_time(options, fstore, statestore):
+    """
+    Will disable any other time synchronization service and configure chrony
+    with given ntp(chrony) server and/or pool using Augeas.
+    If there is no option --ntp-server set IPADiscovery will try to find ntp
+    server in DNS records.
+    """
+    # We assume that NTP servers are discoverable through SRV records in DNS.
+
+    # disable other time&date services first
+    timeconf.force_chrony(statestore)
+
+    print("Synchronizing time")
+    logger.info('Synchronizing time with KDC...')
+
+    if not options.ntp_servers:
+        ds = ipadiscovery.IPADiscovery()
+        ntp_servers = ds.ipadns_search_srv(cli_domain, '_ntp._udp',
```

Comment:
> On a default install no chronyd is configured or started. It is only configured and started if ntp servers or pools are specified.

@rcritten here is attempt to configure chrony if there is any DNS record of NTP server.

Reply:
I just don't understand the inconsistency. Time will not necessarily be kept in sync without chronyd running, right? ntpd uses its default now if no time servers are provided so chronyd should do the same, right? The design doesn't seem to cover this.

Reply:
Right now i understand, your question is about chrony daemon.
chrony as it is has this kind of behavior:
- if we specify --ntp-server or --ntp-pool installer will use this to configure chrony client
- if there is not --ntp-server nor --ntp-pool option specified there is dns lookup for ntp record
- if there is no NTP record installer will not configure chrony

Last step is NOT enabling chrony daemon, so what should ipa-{client,server}-install do?
- [ ] enable chrony and try to sync with default configuration (I prefer this one)
- [ ] end ipa-server-install script with error 

In first case if sync with defaults fails second question is still relevant.

Reply:
I think chronyd should be enabled in all cases inside the client (unless -N is set). You are providing the knobs to be able to control where it syncs to if the end user wants, but time is key to Kerberos, X.509 and MMR.

Reply:
Allrigt, commit added.